content/community/nav.yml

```
@@ -1,42 +1,42 @@
 - title: مصادر المجتمع
   items:
     - id: support
-      title: Support
+      title: الدعم
     - id: courses
-      title: Courses
+      title: الدروس
     - id: examples
-      title: Examples
+      title: الأمثلة
     - id: meetups
-      title: Meetups
+      title: القاءات
     - id: conferences
-      title: Conferences
+      title: المؤتمرات
     - id: articles
-      title: Articles
+      title: المقالات
     - id: podcasts
-      title: Podcasts
+      title: الصوتيات (podcasts)
     - id: videos
-      title: Videos
+      title: الفيديوهات
     - id: external-resources
-      title: External Resources
-- title: أدوات
+      title: المصادر الخارجية
+- title: الأدوات
   items:
     - id: debugging-tools
-      title: Debugging
+      title: التنقيح البرمجة
```

I think that only `التنقيح` word is enough and clear

```suggestion
      title: التنقيح
```

---

التنقيح البرمجي is better !

---

@3imed-jaberi yes, please edit your comment and add it as a suggestion so Ahmed can commit it directly :)